explainer.md

```diff
@@ -151,16 +147,36 @@ partial interface PresentationRequest {
 }
 
 partial interface PresentationConnection {
-  attribute PresentationSource source;
-  [deprecated] readonly attribute USVString url;
+  readonly attribute PresentationSourceType type;
+
+  // Used if `type` is "url".
+  readonly attribute USVString? url;
```

Comment:
I don't think this can change to optional since it's mandatory in the current spec, as that would risk breaking current script.

Reply:
You're right, changed. I suppose it can be an empty string rather than null when not used.